packages/devtools_app/lib/src/memory/flutter/memory_heatmap.dart

```diff
@@ -120,38 +271,33 @@ class FlameChartRenderObject extends RenderBox {
 
   @override
   bool hitTest(BoxHitTestResult result, {Offset position}) {
-    // TODO(terry): Not enabled for testing search.
-    Node foundNode;
-    if (searchString != null) {
-      foundNode = findNode(_sizes.root.children, searchString);
-    }
+    final computedPosition =
+        Offset(position.dx + paintOffset.dx, position.dy + paintOffset.dy);
 
-    if (foundNode == null) {
-      final node = _selectedNode.findRect(position);
-      if (node != null) {
-        selectedNode = node;
-      }
-    } else {
-      // Found the node select it.
-      selectedNode = foundNode;
+    final node = _selectedNode.findRect(computedPosition);
+    if (node != null) {
+      selectedNode = node;
     }
 
     return super.hitTest(result, position: position);
   }
 
   @override
   void paint(PaintingContext context, Offset offset) {
+    paintOffset = offset;
+
     final rootWidth = size.width;
     final top = _paintAncestors(context, _selectedNode.ancestors);
 
-    _paintNode(context, _selectedNode, 0, rootWidth, top);
+    _paintNode(
+        context, _selectedNode, 0 + paintOffset.dx, rootWidth, top - offset.dy);
 
     _paintChildren(
       context: context,
-      currentLeft: 1,
+      currentLeft: 1 + offset.dx,
       parentSize: _selectedNode.byteSize,
       children: _selectedNode.children.values,
-      topFactor: top + 51,
+      topFactor: top + 51 - offset.dy,
```

Comment:
there are a couple constants in this file that are about 50. Make a rowHeight or similar constant that is 50 and use it everywhere so that we can tweak the height without having to track down all occurrences of 50, 51, etc.

Reply:
Must have been old.  There is just one constant nodeHeight = 50 no 51, etc.